homeassistant/components/holiday/config_flow.py

```diff
@@ -0,0 +1,89 @@
+"""Config flow for Holiday integration."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from babel import Locale
+from holidays import list_supported_countries
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_COUNTRY
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.helpers.selector import (
+    CountrySelector,
+    CountrySelectorConfig,
+    SelectSelector,
+    SelectSelectorConfig,
+    SelectSelectorMode,
+)
+
+from .const import CONF_PROVINCE, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+SUPPORTED_COUNTRIES = list_supported_countries(include_aliases=False)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_COUNTRY): CountrySelector(
+            CountrySelectorConfig(
+                countries=list(SUPPORTED_COUNTRIES),
+            )
```

Comment:
We should default it to the setting of the Home Assistant system by default.

Reply:
I just took it from here: https://github.com/home-assistant/core/blob/a1356874ea7cdaea60acdbb797d530309efa633a/homeassistant/components/workday/config_flow.py#L135

Which setting do you mean?

Reply:
I mean the Home Assistant core configuration itself now contains a country field.

Reply:
Yes, but if I use that the user cannot add more calendars for other countries, e.g. friends or collegues. I could use the Home Assistant config as DEFAULT_NAME.

Reply:
Done.